src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -903,22 +908,28 @@ private void UpdateImageMap(PropertyMap fromMap)
 			
             SetProperty(ImageView.Property.IMAGE, new PropertyValue(imageMap));
         }
-
+		
         private void UpdateImage(int key, PropertyValue value)
         {
             PropertyMap temp = new PropertyMap();
 
+            if (_resourceUrl == "")
```

Comment:
could you check the difference of "" and String.Empty ?

Reply:
The result of using String.Empty and "" is the same.
However, I found that String.Empty is more efficient in memory.
https://blog.naver.com/sinjoker/221223223812

I will modify "" to use String.Empty.